flowclient/flowclient/client.py

```diff
@@ -935,3 +935,38 @@ def flows(
         "to_location": to_location,
         "aggregation_unit": aggregation_unit,
     }
+
+
+def unique_subscriber_counts(
+    start_date: str,
+    stop_date: str,
+    aggregation_unit: str,
+) -> dict:
+    """
+    Return query spec for unique subscriber counts
+
+    Parameters
+    ----------
+    start_date : str
+        ISO format date of the first day of the count, e.g. "2016-01-01"
+    end_date : str
+        ISO format date of the day _after_ the final date of the count, e.g. "2016-01-08"
```

Comment:
I think the `end_date` is actually inclusive:
```suggestion
        ISO format date of the final date of the count, e.g. "2016-01-08"
```

Reply:
I entirely agree that it seems more intuitive for a date range to be inclusive. But I copied that comment from another entry point, location_event_counts(). Maybe the idea is that date ranges should be like python substrings, where the "to index" points to the byte after the final one desired?! Can I leave this one for a definitive judgement from someone (Jono?) familiar with the code?

Reply:
It is a slightly whacky legacy thing, because otherwise for a single day you need to write the date twice, or specify the second date in date time form as just before midnight. So yes, end_date is exclusive.

Reply:
Ah thanks, I wasn't aware that that's how things are supposed to work. However, I'm pretty sure this is not the current behaviour because the `end_date` parameters is passed all the way down to the flowmachine library query, which passes it on to `EventTableSubset` where the comparison uses `<=` (see [here](https://github.com/Flowminder/FlowKit/blob/master/flowmachine/flowmachine/features/utilities/event_table_subset.py#L209); or [here](https://github.com/Flowminder/FlowKit/blob/master/flowmachine/flowmachine/features/utilities/event_table_subset.py#L246) for the old version).

In any case, I'm happy to keep the docstring unchanged for this PR and if it needs tweaking we can do this in a separate PR.

Reply:
Yeah, but see https://github.com/Flowminder/FlowKit/blob/master/flowmachine/flowmachine/features/utilities/event_table_subset.py#L157